web-lib-funcs.pl

```diff
@@ -7258,8 +7258,8 @@ sub help_search_link
 if (&foreign_available("man") && !$tconfig{'nosearch'}) {
 	my $for = &urlize(shift(@_));
 	return "<a href='$gconfig{'webprefix'}/man/search.cgi?".
-	       join("&", map { "section=$_" } @_)."&".
-	       "for=$for&exact=1&check=".&get_module_name()."'>".
+	       join("&amp;", map { "section=$_" } @_)."&amp;".
+	       "for=$for&amp;exact=1&amp;check=".&get_module_name()."'>".
```

Comment:
I thought so, too, but not according to the W3C, and I'm guessing they know.

http://htmlhelp.com/tools/validator/problems.html#amp

They're pretty emphatic that & alone is invalid and &amp; is valid.